test/runtests.jl

```diff
@@ -1149,24 +1149,35 @@ end
                         ("'[1:1][-2:-1][3:5]={{{1,2,3},{4,5,6}}}'::float8[]", copyto!(OffsetArray{Union{Missing, Float64}}(undef, 1:1, -2:-1, 3:5), [1 2 3; 4 5 6])),
                         ("'[1:1][-2:-1][3:5]={{{1,2,3},{4,5,6}}}'::oid[]", copyto!(OffsetArray{Union{Missing, LibPQ.Oid}}(undef, 1:1, -2:-1, 3:5), [1 2 3; 4 5 6])),
                         ("'[1:1][-2:-1][3:5]={{{1,2,3},{4,5,6}}}'::numeric[]", copyto!(OffsetArray{Union{Missing, Decimal}}(undef, 1:1, -2:-1, 3:5), [1 2 3; 4 5 6])),
-                        ("'[3,7)'::int4range", Interval(Int32(3), Int32(7), true, false)),
-                        ("'(3,7)'::int4range", Interval(Int32(4), Int32(7), true, false)),
-                        ("'[4,4]'::int4range", Interval(Int32(4), Int32(5), true, false)),
+                        ("'[3,7)'::int4range", Interval{Closed, Open}(Int32(3), Int32(7))),
+                        ("'(3,7)'::int4range", Interval{Closed, Open}(Int32(4), Int32(7))),
+                        ("'[4,4]'::int4range", Interval{Closed, Open}(Int32(4), Int32(5))),
```

Comment:
This seems easier to read to me:
```suggestion
                        ("'[3,7)'::int4range", Interval{Int32, Closed, Open}(3, 7)),
                        ("'(3,7)'::int4range", Interval{Int32, Closed, Open}(4, 7)),
                        ("'[4,4]'::int4range", Interval{Int32, Closed, Open}(4, 5)),
```